admin/data/sql/migrations/20190110132657.do.create-view-check-windows-with-status-and-form-count-by-form-type.sql

```diff
@@ -0,0 +1,41 @@
+CREATE VIEW [mtc_admin].[vewCheckWindowsWithStatusAndFormCountByFormType] AS
```

Comment:
Form Count By Form Type - I suspect we could drop the second 'form' and just call it `vewCheckWindowsWithStatusAndFormCountByType`